src/app/login/login.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container-fluid">
+  <div class="row">
+    <div class="col-md-6 d-flex align-items-center justify-content-center">
+
+      <form [formGroup]="logForm" (ngSubmit)="onSubmit()">
+
+        <div class="text-center">
+          <h2>LOGIN</h2>
+          <p class="text-secondary">Please enter your credentials</p>
+        </div>
+
+        <div class="p-1">
+          <div class="error-message" *ngIf="submitted">
```

Comment:
```suggestion
          <div class="error-message" *ngIf="submitted && showErrorMessage">
```

Reply:
I added this code plus the function below.
the page is not behaving correctly

Reply:
Apologies,  updated get: 
`get showErrorMessage(): boolean {`
   ` return this.logForm.controls['username'].errors !== null`
     ` || this.logForm.controls['password'].errors !== null`
`  }`